administrator/templates/atum/scss/pages/_com_cpanel.scss

```
@@ -174,6 +174,10 @@
     &:hover {
       background: var(--toolbar-bg);
     }
+    
+   a[class^="flex-grow-1"] {
+     text-decoration: underline;
```

```suggestion
    a[class^="flex-grow-1"] {
      text-decoration: underline;
```
Please indent in the same way as you can see in previous code parts. Code style is important for quick readability and so maintainability of source code.